tests/models/tvp/test_processor_tvp.py

```diff
@@ -0,0 +1,75 @@
+import inspect
+import tempfile
+import unittest
+
+from transformers import TvpProcessor
+from transformers.testing_utils import require_torch, require_vision
+
+from ...test_processing_common import ProcessorTesterMixin
+
+
+class TvpProcessorTest(ProcessorTesterMixin, unittest.TestCase):
+    from_pretrained_id = "Jiqing/tiny-random-tvp"
+    processor_class = TvpProcessor
+    videos_data_arg_name = "pixel_values"
```

Comment:
@zucchini-nlp just a heads up cuz I remember you mentioning that you plan on reworking the text-videos models:

the output keys for video data is inconsistent; some uses `pixel_values_videos` while some just uses `pixel_values`. You'd most likely need to uniformize/standardize them too

Reply:
Yes, it's me who started using `pixel_values_videos` hehe, thanks for letting know :)